utopian-io/v2.utopian.io
# Handle authenticated page in the router

### Verify that

### **Test plan**

_SSR test_

* Log out from the app
* Go to http://localhost:8080/en/projects/create
* Complete the login flow

**Expected result:** You should be redirected to the login page and then back to the create project page

_In app test_

* Log out from the app
* Click on the submit project button on the homepage
* Complete the login flow

**Expected result:** You should be redirected to the login page and then back to the create project page

NB: there are some JS errors coming from the logout, it doesn't affect this PR. It will be dealt with separately
NB1: I can't reproduce this...
